src/routes/choose-outfit/+page.svelte

```diff
@@ -36,14 +46,15 @@
     {:else}
         <div class="display_outfit">
             <div class="box">
-                <Header type="h2">Outfit Top: </Header>
-
+                <Header type="h2">Outfit Top: </Header> <br>
+                <img src={presetClothes[0].img} alt="shirt">
```

Comment:
You shouldn't need to use presetClothes at all here. They are preloaded into the store when the store is created. You should just be able to do something like 
`clothesStore.getClothingItemById(outfitStore[0].topid).img`